### PR TITLE
ref(ui): Consistently use SentryDocumentTitle

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/components/sentryDocumentTitle.tsx [hunk 1]
```diff
@@ -22,11 +42,20 @@ function SentryDocumentTitle({title, orgSlug, projectSlug, children}: Props) {
       return `${title} - ${orgSlug}`;
     }
 
-    return `${title} - ${projectSlug}`;
+    if (projectSlug) {
+      return `${title} - ${projectSlug}`;
+    }
+
+    return title;
```
Comment (evanpurkhiser) [inline static/app/components/sentryDocumentTitle.tsx:49]: Just changed around the logic here a bit to make it more clear

File: static/app/views/sharedGroupDetails/index.tsx [hunk 2]
```diff
@@ -107,7 +103,7 @@ class SharedGroupDetails extends Component<Props, State> {
     const title = this.getTitle();
 
     return (
-      <DocumentTitle title={title}>
+      <SentryDocumentTitle noSuffix title={title}>
```
Comment (evanpurkhiser) [inline static/app/views/sharedGroupDetails/index.tsx:106]: Group titles historically don't have `- Sentry` at the end.

File: static/app/views/settings/settingsIndex.tsx [hunk 3]
```diff
@@ -80,7 +80,9 @@ class SettingsIndex extends React.Component<Props> {
     const supportText = isOnPremise ? t('Community Forums') : t('Contact Support');
 
     return (
-      <DocumentTitle title={organization ? `${organization.slug} Settings` : 'Settings'}>
+      <SentryDocumentTitle
+        title={organization ? `${organization.slug} Settings` : 'Settings'}
```
Comment (priscilawebdev) [inline static/app/views/settings/settingsIndex.tsx:84]:
```suggestion
        title={t('Settings')}
        orgSlug={organization?.slug}
```

Comment (evanpurkhiser) [reply static/app/views/settings/settingsIndex.tsx:84]: This one might make sense to keep as `my-org Settings - Sentry` maybe? vs `Settings - my-org - Sentry`

Comment (evanpurkhiser) [reply static/app/views/settings/settingsIndex.tsx:84]: maybe not, I might agree here

File: static/app/components/sentryDocumentTitle.tsx [hunk 4]
```diff
@@ -2,18 +2,38 @@ import * as React from 'react';
 import DocumentTitle from 'react-document-title';
 
 type Props = {
+  /**
+   * This string will be shown at the very front of the title
+   */
   title?: string;
+  /**
+   * The project slug to show in the title
```
Comment (markstory) [inline static/app/components/sentryDocumentTitle.tsx:10]:
```suggestion
   * The organization slug to show in the title
```

**Review by markstory — APPROVED** [review]

File: static/app/views/settings/settingsIndex.tsx (hunk 3 above)
Comment (evanpurkhiser) [reply static/app/views/settings/settingsIndex.tsx:84]: Will get this in a 2nd pr

**Pull request merged**